docs/articles/call-settings.md

```diff
@@ -1,3 +1,61 @@
 # Call settings
 
-(Chris Bacon to write this)
\ No newline at end of file
+## Introduction
+
+All client libraries wrapping gRPC-based APIs (see [api-layers](api-layers.md))
+allow customization of RPC calls by using
+[CallSettings](../obj/api/Google.Api.Gax.CallSettings.yml). If `CallSettings` are
+not specified sensible defaults are automatically provided.
+See the [CallSettings documentation](../obj/api/Google.Api.Gax.CallSettings.yml)
+for descriptions of the available properties.
+
+The underlying gRPC API uses
+[CallOptions](http://www.grpc.io/grpc/csharp/html/T_Grpc_Core_CallOptions.htm),
+which provides a slightly reduced feature set compared to `CallSettings`.
+Client libraries automatically construct a `CallOptions` from `Callsettings`
+to `CallOptions` as required.
+
+There are three distinct places where `CallSettings` may optionally be provided:
+
+* When calling an RPC method *(highest priority)*.
+* Client-wide when creating a client object (for example, using
+[PublisherClient.Create()](../obj/api/Google.Pubsub.V1.PublisherClient.html#Google_Pubsub_V1_PublisherClient_Create_Google_Api_Gax_ServiceEndpoint_Google_Pubsub_V1_PublisherSettings_)).
+* Per-RPC-method when creating a client object *(lowest priority)*.
+
+It's important to understand exactly *which* properties of which `CallSettings` will be used during an RPC invocation.
```

Comment:
Let's combine the next paragraph with this one - they're about the same thing. I'm also not sure about the emphasis of "which".